google/cloud/spanner_dbapi/cursor.py

```diff
@@ -335,7 +340,9 @@ def fetchmany(self, size=None):
         items = []
         for i in range(size):
             try:
-                items.append(tuple(self.__next__()))
```

Comment:
Can't say why `tuple` was used here. It doesn't make sense to me, so I assume it was just a logical error: in PEP 249 they mention returning tuples in this method as an example. But it is just an example.